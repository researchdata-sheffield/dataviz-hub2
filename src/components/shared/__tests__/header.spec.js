/**
 * @jest-environment jsdom
 */
/* eslint-disable no-unused-vars */
/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import Header from "../header";
import { LocationProvider } from "@gatsbyjs/reach-router";
import { render, fireEvent } from "@testing-library/react";

describe("Header", () => {
  it("renders correctly", async () => {
    let tree = renderer
      .create(
        <LocationProvider>
          <Header />
        </LocationProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("Header Interaction", () => {
  let header;
  document.execCommand = jest.fn();

  beforeEach(() => {
    header = render(
      <LocationProvider>
        <Header />
      </LocationProvider>
    );
  });

  it("open the dialog box when click RSS button (desktop)", async () => {
    const rssDesktop = await header.container.querySelector("#rssDesktop");
    await fireEvent.click(rssDesktop);

    const rssBox = await header.container.querySelector("#rssBox");

    expect(rssBox.style.visibility).toBeFalsy();
    expect(document.execCommand).toHaveBeenCalledWith("copy");
  });

  it("open the dialog box when click RSS button (mobile)", async () => {
    Object.defineProperty(window, "innerWidth", {
      writable: true,
      configurable: true,
      value: 400
    });

    await window.dispatchEvent(new Event("resize"));

    const rssMobile = await header.container.querySelector("#rssMobile");
    await fireEvent.click(rssMobile);

    const rssBox = await header.container.querySelector("#rssBox");
    expect(rssBox.style.visibility).toBeFalsy();
    expect(document.execCommand).toHaveBeenCalledWith("copy");
  });
});
