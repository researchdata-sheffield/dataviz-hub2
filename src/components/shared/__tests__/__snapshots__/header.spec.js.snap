// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="font-semibold z-50 relative group"
>
  <nav
    className="group-hover:bg-white bg-transparent transition duration-500 flex items-center justify-between flex-wrap px-5 fixed w-full z-10 overflow-hidden"
    id="navbar"
    role="navigation"
    style={
      Object {
        "transition": "top 0.3s",
      }
    }
  >
    <div
      className="flex items-center flex-shrink-0 mr-5"
    >
      <a
        className="sc-crHmcD ixmMsE invisible "
        href="https://www.sheffield.ac.uk/"
        rel="noopener noreferrer"
        target="_blank"
        title="The University of Sheffield Logo"
      >
        <img
          alt="The University of Sheffield Logo"
          className="mt-1"
          src="test-file-stub"
          style={
            Object {
              "maxWidth": "13.6vh",
            }
          }
        />
      </a>
      <div
        className="invisible ml-4 text-lg font-extrabold transition duration-1000 ease-in-out"
      >
        <a
          className="textanimate"
          gatsby="true"
          href="/"
        >
          Dataviz.Shef
        </a>
      </div>
    </div>
    <div
      className="block xl:hidden py-2"
      onClick={[Function]}
      style={
        Object {
          "zIndex": "999",
        }
      }
    >
      <button
        className="px-3 py-2 flex items-center border rounded outline-none transition duration-300 group-hover:border-gray-700 group-hover:text-gray-700 text-gray-700 border-gray-700"
        title="Menu"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
          />
        </svg>
      </button>
    </div>
    <div
      className="invisible opacity-0 translate-x-full transform rounded-md xl:hidden shadow-lg px-6 py-10 md:py-16 fixed top-0 left-0 h-full min-w-full overflow-y-scroll"
      id="mobileHeader"
      style={
        Object {
          "backgroundColor": "#fbfbfb",
          "transition": ".3s linear",
          "zIndex": "990",
        }
      }
    >
      <div
        className="mt-5 justify-end flex flex-wrap items-center"
      >
        <a
          className="transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/"
          mobile="true"
          type="Link"
        >
          Home
        </a>
        <a
          className="transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/community"
          mobile="true"
          type="Link"
        >
          Community
        </a>
        <a
          className="transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/events"
          mobile="true"
          type="Link"
        >
          Events
        </a>
        <a
          className="transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/blog"
          mobile="true"
          type="Link"
        >
          Blog
        </a>
        <a
          className="sc-crHmcD ixmMsE transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          href="https://orda.shef.ac.uk/visualisations/"
          rel="noopener noreferrer"
          style={
            Object {
              "background": "none",
            }
          }
          target="_blank"
          type="a"
        >
          Showcase
        </a>
        <a
          className="transition duration-300 text-gray-600 hover:text-brand-blue block w-full text-2xl transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/about"
          mobile="true"
          type="Link"
        >
          About
        </a>
        <div
          className="flex flex-wrap w-full justify-center space-x-4 mt-8 pt-6 border-t-2 border-gray-200"
        >
          <a
            className="sc-crHmcD ixmMsE transition duration-300 text-gray-600 hover:text-brand-blue block mt-3 flex items-center"
            href="https://groups.google.com/a/sheffield.ac.uk/g/shef_dataviz-group"
            rel="noopener noreferrer"
            style={
              Object {
                "background": "none",
              }
            }
            target="_blank"
            title="Join Google group"
            type="a-icon"
          >
            <div
              className="px-4 text-2xl transition duration-500 ease-in-out no-underline"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 488 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"
                />
              </svg>
            </div>
          </a>
          <a
            className="sc-crHmcD ixmMsE transition duration-300 text-gray-600 hover:text-brand-blue block mt-3 flex items-center"
            href="https://shef-dataviz.slack.com/join/signup#/domain-signup"
            rel="noopener noreferrer"
            style={
              Object {
                "background": "none",
              }
            }
            target="_blank"
            title="Join Slack channel"
            type="a-icon"
          >
            <div
              className="px-4 text-2xl transition duration-500 ease-in-out no-underline"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"
                />
              </svg>
            </div>
          </a>
          <a
            className="transition duration-300 text-gray-600 hover:text-brand-blue block mt-3 flex items-center"
            gatsby="true"
            id="rssMobile"
            mobile="true"
            onClick={[Function]}
            target="_self"
            title="RSS Feed"
            type="Link-icon"
          >
            <button
              className="relative px-4 text-2xl transition duration-500 ease-in-out no-underline"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"
                />
              </svg>
              <span
                className="invisible group-hover:visible absolute top-0 right-0 bg-red-500 text-white rounded-md text-xs -mt-4 -mr-2"
                style={
                  Object {
                    "padding": "2px 5px",
                  }
                }
              >
                new
              </span>
            </button>
          </a>
          <a
            className="transition duration-300 text-gray-600 hover:text-brand-blue block mt-3 flex items-center"
            gatsby="true"
            href="/search"
            mobile="true"
            title="search dataviz hub"
            type="Link-icon"
          >
            <div
              className="px-4 text-2xl transition duration-500 ease-in-out no-underline"
            >
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="11"
                  cy="11"
                  r="8"
                />
                <line
                  x1="21"
                  x2="16.65"
                  y1="21"
                  y2="16.65"
                />
              </svg>
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="hidden py-2 xl:flex items-center w-auto text-xs xl:text-sm"
      id="desktopHeader"
    >
      <div
        className="justify-end flex flex-wrap items-center"
      >
        <a
          className="transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/community"
          type="Link"
        >
          Community
        </a>
        <a
          className="transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/events"
          type="Link"
        >
          Events
        </a>
        <a
          className="transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/blog"
          type="Link"
        >
          Blog
        </a>
        <a
          className="sc-crHmcD ixmMsE transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          href="https://orda.shef.ac.uk/visualisations/"
          rel="noopener noreferrer"
          style={
            Object {
              "background": "none",
            }
          }
          target="_blank"
          type="a"
        >
          Showcase
        </a>
        <a
          className="transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined transition duration-500 ease-in-out md:mr-2 no-underline px-4 py-1"
          gatsby="true"
          href="/about"
          type="Link"
        >
          About
        </a>
        <a
          className="sc-crHmcD ixmMsE transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined flex items-center"
          href="https://groups.google.com/a/sheffield.ac.uk/g/shef_dataviz-group"
          rel="noopener noreferrer"
          style={
            Object {
              "background": "none",
            }
          }
          target="_blank"
          title="Join Google group"
          type="a-icon"
        >
          <div
            className="text-md transition duration-500 ease-in-out inline-block no-underline ml-2 mr-2 -mb-1"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 488 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"
              />
            </svg>
          </div>
        </a>
        <a
          className="sc-crHmcD ixmMsE transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined flex items-center"
          href="https://shef-dataviz.slack.com/join/signup#/domain-signup"
          rel="noopener noreferrer"
          style={
            Object {
              "background": "none",
            }
          }
          target="_blank"
          title="Join Slack channel"
          type="a-icon"
        >
          <div
            className="text-md transition duration-500 ease-in-out inline-block no-underline ml-2 mr-2 -mb-1"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"
              />
            </svg>
          </div>
        </a>
        <a
          className="sc-crHmcD ixmMsE transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined flex items-center"
          id="rssDesktop"
          onClick={[Function]}
          rel="noopener noreferrer"
          style={
            Object {
              "background": "none",
            }
          }
          target="_blank"
          title="RSS Feed"
          type="a-icon"
        >
          <button
            className="relative text-md transition duration-500 ease-in-out inline-block no-underline ml-2 mr-2 -mb-1 cursor-pointer"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"
              />
            </svg>
            <span
              className="invisible group-hover:visible absolute top-0 right-0 bg-red-500 text-white rounded-md -mt-4 -mr-4"
              style={
                Object {
                  "fontSize": ".7rem",
                  "lineHeight": "1rem",
                  "padding": "1px 3px",
                }
              }
            >
              new
            </span>
          </button>
        </a>
        <a
          className="transition duration-300 text-gray-100 group-hover:text-black hover:text-brand-blue undefined flex items-center"
          gatsby="true"
          href="/search"
          title="Search"
          type="Link-icon"
        >
          <div
            className="mr-4 ml-2 text-sm xl:text-lg transition duration-500 ease-in-out md:mr-2 inline-block no-underline"
          >
            <svg
              fill="none"
              height="1em"
              stroke="currentColor"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="11"
                cy="11"
                r="8"
              />
              <line
                x1="21"
                x2="16.65"
                y1="21"
                y2="16.65"
              />
            </svg>
          </div>
        </a>
      </div>
    </div>
    <div
      className="invisible fixed bg-white shadow-2xl rounded-lg p-5 lg:p-10 text-center"
      id="rssBox"
      style={
        Object {
          "left": "50%",
          "top": "50%",
          "transform": "translate(-50%, -50%)",
          "zIndex": "1000",
        }
      }
    >
      <svg
        className="text-2xl absolute top-0 right-0 text-gray-800 m-2 rounded-full hover:text-white hover:bg-red-500 transition duration-300 p-1"
        fill="currentColor"
        height="1em"
        onClick={[Function]}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 352 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
        />
      </svg>
      <svg
        className="text-5xl mx-auto"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M128.081 415.959c0 35.369-28.672 64.041-64.041 64.041S0 451.328 0 415.959s28.672-64.041 64.041-64.041 64.04 28.673 64.04 64.041zm175.66 47.25c-8.354-154.6-132.185-278.587-286.95-286.95C7.656 175.765 0 183.105 0 192.253v48.069c0 8.415 6.49 15.472 14.887 16.018 111.832 7.284 201.473 96.702 208.772 208.772.547 8.397 7.604 14.887 16.018 14.887h48.069c9.149.001 16.489-7.655 15.995-16.79zm144.249.288C439.596 229.677 251.465 40.445 16.503 32.01 7.473 31.686 0 38.981 0 48.016v48.068c0 8.625 6.835 15.645 15.453 15.999 191.179 7.839 344.627 161.316 352.465 352.465.353 8.618 7.373 15.453 15.999 15.453h48.068c9.034-.001 16.329-7.474 16.005-16.504z"
        />
      </svg>
      <h2
        className="text-2xl font-bold mt-4"
      >
        RSS Feed link copied!
      </h2>
      <h6
        className="text-gray-600 text-sm mt-1"
      >
        Doesn't work? Use the following link
      </h6>
      <div
        className="inline-block bg-white shadow rounded-md mt-5"
      >
        <input
          className="py-2 pl-2 text-base focus:outline-none pr-3 text-gray-500"
          id="rssLink"
          readOnly={true}
          type="text"
          value="https://dataviz.shef.ac.uk/rss.xml"
        />
        <button
          className="bg-brand-pink p-2 rounded-md cursor-pointer"
          id="copyBtn"
          onClick={[Function]}
        >
           
          COPY
           
        </button>
      </div>
    </div>
  </nav>
</header>
`;
